control-plane/pkg/reconciler/source/v2/sourcev2_test.go

```diff
@@ -774,6 +816,13 @@ func TestReconcileKind(t *testing.T) {
 	}))
 }
 
+func StatusSourceSelector(srcName string) KRShapedOption {
+	return func(obj duckv1.KRShaped) {
+		ks := obj.(*sources.KafkaSource)
+		ks.Status.Selector = "eventing.knative.dev/source=kafka-source-controller,eventing.knative.dev/sourceName=" + srcName
+	}
+}
```

Comment:
Can we use this approach?

```suggestion
func StatusSourceSelector() KRShapedOption {
	return func(obj duckv1.KRShaped) {
		ks := obj.(*sources.KafkaSource)
		ks.Status.Selector = "eventing.knative.dev/source=kafka-source-controller,eventing.knative.dev/sourceName=" + ks.Name
	}
}
```